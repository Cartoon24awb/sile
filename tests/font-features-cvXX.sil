\begin[papersize=a5]{document}
\script[src=packages/features]
\set[parameter=harfbuzz.subshapers,value=ot]
\neverindent\nofolios
\par\font[CharacterVariant="{75,81}"]{Hoa hồng đẹp. «Һ» хәрефе төрки һәм...
\par\font[CharacterVariant=No75]{Hoa hồng đẹp}. \font[CharacterVariant=No81]{«Һ» хәрефе төрки һәм...}
\par\font[CharacterVariant="{No75, No81}"]{Hoa hồng đẹp. «Һ» хәрефе төрки һәм...}}
\end{document}
