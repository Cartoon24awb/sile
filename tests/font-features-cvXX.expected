Set paper size 	419.5275636	595.275597
Begin page
Mx 	20.9764
My 	37.7423
Set font 	Gentium Plus;10;400;;normal;+cv75;+cv81;LTR
T	43 82 68	(Hoa)
Mx 	39.7994
T	75 1307 81 74	(hồng)
Mx 	63.4076
T	559 639 83	(đẹp)
Mx 	78.5248
T	17	(.)
Mx 	83.4807
T	169	(«)
Mx 	88.3342
T	903	(Һ)
Mx 	94.3108
T	170	(»)
Mx 	101.8301
T	1831 651 1430 611 1453 611	(хәрефе)
Mx 	135.1991
T	1641 1334 1430 1081 1259	(төрки)
Mx 	163.7487
T	859 651 1211	(һәм)
Mx 	180.5114
T	17	(.)
Mx 	182.8014
T	17	(.)
Mx 	185.0915
T	17	(.)
Mx 	20.9764
My 	50.6515
Set font 	Gentium Plus;10;400;;normal;+cv75;+cv81;-cv75;+cv81;LTR
T	43 82 68	(Hoa)
Mx 	39.7994
T	75 1306 81 74	(hồng)
Mx 	63.4076
T	559 639 83	(đẹp)
Mx 	78.5248
Set font 	Gentium Plus;10;400;;normal;+cv75;+cv81;LTR
T	17	(.)
Mx 	83.4806
Set font 	Gentium Plus;10;400;;normal;+cv75;+cv81;+cv75;-cv81;LTR
T	169	(«)
Mx 	88.3341
T	903	(Һ)
Mx 	94.3107
T	170	(»)
Mx 	101.8300
T	1831 651 1430 611 1453 611	(хәрефе)
Mx 	135.1989
T	1641 1334 1430 1081 1259	(төрки)
Mx 	163.7485
T	858 651 1211	(һәм)
Mx 	180.8139
T	17	(.)
Mx 	183.1040
T	17	(.)
Mx 	185.3940
T	17	(.)
Mx 	20.9764
My 	63.5607
Set font 	Gentium Plus;10;400;;normal;+cv75;+cv81;-cv75;-cv81;LTR
T	43 82 68	(Hoa)
Mx 	39.7994
T	75 1306 81 74	(hồng)
Mx 	63.4076
T	559 639 83	(đẹp)
Mx 	78.5248
T	17	(.)
Mx 	83.4806
T	169	(«)
Mx 	88.3341
T	903	(Һ)
Mx 	94.3107
T	170	(»)
Mx 	101.8300
T	1831 651 1430 611 1453 611	(хәрефе)
Mx 	135.1989
T	1641 1334 1430 1081 1259	(төрки)
Mx 	163.7485
T	858 651 1211	(һәм)
Mx 	180.8139
T	17	(.)
Mx 	183.1040
T	17	(.)
Mx 	185.3940
T	17	(.)
End page
Finish
